test/unit/controllers/declaration/CommodityDetailsControllerSpec.scala

```diff
@@ -69,6 +72,8 @@ class CommodityDetailsControllerSpec extends ControllerSpec with OptionValues {
 
       "display page method is invoked and cache is empty" in {
 
+        withNewCaching(aDeclaration())
```

Comment:
If we would merge earlier #662 there are helper methods for testing controllers against multiple declaration type.

Reply:
We can merge it before, but I'll prefer to stay with this solution for now to finish ticket, we need to refactor a lot of controller specs, so it can be done in separate PR